### PR TITLE
Update Installation instructions

### PR DESCRIPTION
**Pull request by wabash1**
Moved installation to top, added quick install instructions for comma 3, switched URL from Smiskol to installer.comma.ai

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,33 @@ The following changes are a **VIOLATION** and **ARE NOT** included:
...
+To install sunnyhaibin's fork, simply use the installer.comma.ai URL (thanks [Shane](https://github.com/sshane/openpilot-installer-generator)!) on the setup screen for "Custom Software" after you factory reset or uninstalled sunnypilot from a previous install:
+
+```
+installer.comma.ai/sunnyhaibin/<insert_branch_name>
+```
+For example, if you would like to install the branch for HKG:
+
+* [`0.8.12-prod-full-hkg`](https://github.com/sunnyhaibin/openpilot/tree/0.8.12-prod-full-hkg):
+    ```
+    installer.comma.ai/0.8.12-prod-full-hkg
```
Comment (Moodkiller) [inline README.md:70]: This should be: `installer.comma.ai/sunnyhaibin/0.8.12-prod-full-hkg`

File: README.md [hunk 2]
```diff
@@ -48,6 +48,33 @@ The following changes are a **VIOLATION** and **ARE NOT** included:
   * ❌ Re-engaging of <ins>**LONGITUDINAL CONTROL**</ins> (acceleration/brake) on brake pedal release ❌ (was never included in MADS)
   * ❌ No disengaging on ACC MAIN in OFF state ❌
 
+⚒ Installation
+---
+
+### Comma URL (Comma 3 only quick setup)
+* Test-C3 is more stable than 8.14
```
Comment (Moodkiller) [inline README.md:55]: Test-C3 -> `test-c3` (encapsulate in ` ). Also I would actually remove this line or swap it around as `test-c3` is _less_ stable compared to 0.8.14 IMO.

Comment (Moodkiller) [reply README.md]: No problem! Sounds good and looks good. Thanks for the revisions

File: README.md [hunk 3]
```diff
@@ -48,6 +48,33 @@ The following changes are a **VIOLATION** and **ARE NOT** included:
...
   * ❌ No disengaging on ACC MAIN in OFF state ❌
 
+⚒ Installation
+---
+
+### Comma URL (Comma 3 only quick setup)
+* 8.14 was last updated in July 2022, many changes have been made since then so test-c3 is recommended
+* Factory reset/uninstall previous software if you already installed a branch
+* Select Custom Software
+* Input the following URL: ```installer.comma.ai/sunnyhaibin/test-c3```
```
Comment (sunnyhaibin) [inline README.md:58]: Thanks for the review @Moodkiller !

@wabash1 Can we list these steps as numerical orders such as the one mentioned in "[Dynamic Lane Profile](https://github.com/sunnyhaibin/sunnypilot#dynamic-lane-profile-dlp)"?

**Comment by sunnyhaibin** [comment]
LGTM. Thanks for the PR!

**Pull request merged**